src/context/FdpStorageContext.tsx

```diff
@@ -29,7 +30,7 @@ interface FdpStorageContextProps {
 }
 
 interface FdpStorageContext {
-  fdpClient: FdpStorage;
+  fdpClient: any;
```

Comment:
why ? we loose type, this is not good solution. What needs to be fixed in order to use proper type ?